ITR/interfaces.py

```diff
@@ -393,7 +467,7 @@ def pint_ify(x, units='dimensionless'):
         if x.startswith('nan '):
             return Q_(np.nan, units)
         return ureg(x)
-    if isinstance(x, pint.Quantity):
+    if isinstance(x, Quantity):
```

Comment:
we have inconsistency across classes.
some of them use `pint.Quantity`, and some use `Quantity`.
should we try to stick to the same style across the project?

Reply:
It's the source code you are looking at that's inconsistent.  We are using pint.Quantity everywhere.  Again, I have no idea how this is happening.

Reply:
i see, all right